db/migrate/20160324174132_create_container_deployments.rb

```diff
@@ -0,0 +1,25 @@
+class CreateContainerDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string     :deployment_type
+      t.string     :deployment_version
+      t.boolean    :containerized, :default => true
+      t.string     :deployment_method
+      t.string     :ssh_private
+      t.string     :ssh_public
+      t.string     :ssh_user
+      t.string     :rhsm_user
+      t.string     :rhsm_pass
+      t.string     :rhsm_sku
+      t.string     :rhsm_pool_id
+      t.string     :rhsm_server
+      t.string     :rhsm_activation_key
+      t.string     :metrics_endpoint
+      t.text       :customize
+      t.belongs_to :automation_task, :type => :bigint
+      t.belongs_to :deployed_ems,    :type => :bigint
+      t.belongs_to :deployed_on_ems, :type => :bigint
```

Comment:
I had the same problem, this was the explanation:

> Sure, so the deployed_ems is the openshift/atomic ems that you are creating by using the deployment feature, the deployed_on_ems is either the provider you provision vms on and then deploy the cluster or the provider that you choose existing vms to deploy on.

I dont have a better naming, but this could be indeed more self explanatory.